src/AppStateContext.tsx

```diff
@@ -298,6 +296,23 @@ export const AppStateProvider = ({ children }: React.PropsWithChildren<{}>) => {
         }
       );
 
+      //Get the history on the default channel.
+      state.pubnub.history(
+          {
+              channel: state.channel,
+              count: 100
+          },
+          (status, response) => {
+            if (typeof response.messages !== "undefined" && response.messages.length > 0) {
+              for (var i = 0; i <= response.messages.length; i++) {
+                dispatch({
+                  type: "ADD_MESSAGE",
```

Comment:
Use ADD_HISTORY instead that was added on the main branch.

Reply:
Then if we use ADD_MESSAGE we want to make sure that we have an option to turn history off and when we do we want to make sure we still subscribe which is not the case here.